src/werkzeug/debug/shared/debugger.js

```diff
@@ -11,7 +11,11 @@ docReady(function() {
     addConsoleIconToFrames(document.querySelectorAll('div.traceback div.frame'));
     addToggleTraceTypesOnClick(document.querySelectorAll('h2.traceback'));
     addInfoPrompt(document.querySelectorAll('span.nojavascript'));
-    plainTraceback(document.querySelectorAll('div.plain, textarea'));
+    const plainTraceback = document.querySelector("div.plain textarea");
+    const wrapper = document.createElement("pre");
+    const textNode = document.createTextNode(plainTraceback.textContent);
+    wrapper.appendChild(textNode);
+    plainTraceback.replaceWith(wrapper);
```

Comment:
Can we move lines 14-18 into its own function?

Reply:
I agree. Our convention so far has been to abstract everything into functions for readability (abstract this into a function called `plainTraceback()`?), other than that it looks great! 👍

Reply:
I agree. Our convention so far has been to abstract everything into functions for readability (abstract this into a function called `plainTraceback()`?), other than that it looks great! 👍